src/embed/types.rs

```diff
@@ -132,6 +132,8 @@ pub struct ScanOptions {
     pub prefix: Option<String>,
     pub start: Option<ScanBound>,
     pub limit: Option<u64>,
+    #[serde(rename = "indexName")]
```

Comment:
I know it is too late but i wish we had used `snake_case` for our JSON

Reply:
It's not too late! This is private API. We can just change it and change the client in lockstep.

Reply:
(and I agree)

Reply:
This makes more sense as a separate PR. Agree we should do it but not doing it as part of this.